app/controllers/users_controller.rb

```diff
@@ -1,18 +1,20 @@
 class UsersController < ApplicationController
 
   def show
-    # require 'pry'; binding.pry
-    # user = User.find_by(user_params[:id])
-  
-    conn = Faraday.new("https://api.github.com") do |req|
-      req.headers["authorization"] = ENV["GH_API_KEY"] 
+    @user = User.find_by(params[:user_id])
+    if @user.token == nil
+      []
+    else
+      conn = Faraday.new("https://api.github.com") do |req|
+        req.headers["authorization"] = @user.token
```

Comment:
Now because token is in the db we are able to call it without having the api key visibile

Reply:
Nice, that is a great idea.